### PR TITLE
COMMERCE-10750

### PR DESCRIPTION
**Pull request by Brunno0liveira**
https://issues.liferay.com/browse/COMMERCE-10750

### CODE REVIEW

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 1]
```diff
@@ -2222,6 +2222,138 @@ definition {
...
+				productName = "U-Joint",
+				productQuantity = 2,
+				productUnitPrice = 24,
+				shippingMethod = "Standard");
+		}
+
+		task ("And the Order SKU is searched on the inventory page") {
+			CommerceNavigator.gotoPortlet(
+				category = "Inventory Management",
+				portlet = "Inventory");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2310]: Please use `CommerceInventory.openInventoryAdmin();`

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 2]
```diff
@@ -2222,6 +2222,138 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-5845. As a product specialist I want to see if a SKU is on order"
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2225]: Please update the description

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by lorenzolapescara** [comment]
@Brunno0liveira one more thing: can you please move the Autogenerated commits to be the last ones? Thanks

**Comment by Brunno0liveira** [comment]
ci:test:relevant

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase [hunk 3]
```diff
@@ -2222,6 +2222,123 @@ definition {
...
+		task ("Given a Minium site with money order enable and two accounts are created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceChannels.openChannelsAdmin();
+
+			CommerceNavigator.gotoEntry(entryName = "Minium Portal");
+
+			CommerceChannels.editPaymentMethod(
+				active = "true",
+				paymentMethod = "Money Order");
```
Comment (lorenzolapescara) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceOrders.testcase:2239]: @Brunno0liveira as this payment method was enabled in the setup, at this point in the test Money Order is already enabled, please remove these lines

**Review by lorenzolapescara — CHANGES_REQUESTED** [review]

**Comment by Brunno0liveira** [comment]
ci:test:sf

**Comment by Brunno0liveira** [comment]
ci:test:relevant

**Comment by baharturk** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:test:relevant

**Comment by lorenzolapescara** [comment]
ci:stop

**Comment by lorenzolapescara** [comment]
Re-sent at #3504.

**Comment by lorenzolapescara** [comment]
ci:close

**Pull request closed without merging**